cypress/integration/percentile-calculator-page.spec.ts

```diff
@@ -36,6 +36,7 @@ describe('Percentile Calculator Page', () => {
   describe('For analytics', () => {
     it('Some DOM elements should exist with IDs', () => {
       cy.visit('/percentile-calculator/')
+      cy.get('#disclaimer-accept-button')
```

Comment:
You could do `cy.get('#disclaimer-accept-button').click()` and remove the next line